### DISCUSSION TITLE
unable to send user data to callbackurl

### DISCUSSION THREAD

**Discussion by sumanthreddy-a** [jaredhanson/passport#816] [Help]
expressjs 4.0
nodejs 10
passportjs 0.4
passport-facebook 3.0
passport-google-oauth2

passport.use(new GoogleStrategy({
    clientID: config.google.clientID,
    clientSecret: config.google.clientSecret,
    callbackURL: config.google.callbackURL,
    scope: ['profile', 'email'],
    passReqToCallback: true
},
    async function (request, accessToken, refreshToken, profile, done) {
        try {
//user create or check

}catch(err){}})

//callback url
router.get('/google/callback',
    passport.authenticate('google', {
        session: false,
        successRedirect: `${config.google.successRedirect}`,
        failureRedirect: config.google.failureRedirect,
    })
)

unable to send user data to callback URL

can anyone help me out this

**Comment by gabgagnon** [comment]
https://github.com/jaredhanson/passport/issues/570#issuecomment-638644317

Maybe it will help you?
